Resolution/Resolution.Tests/VisitorsTests/ClauseMakerVisitorTests.cs

```diff
@@ -59,15 +59,15 @@ public void ConjunctionOfClausesTest()
             Assert.AreEqual(2, clauseCollection.Count);
 
             Assert.AreEqual(1, clauseCollection[0].PositiveLiterals.Count);
-            Assert.AreEqual(p, clauseCollection[0].PositiveLiterals[0]);
+            Assert.IsTrue(clauseCollection[0].PositiveLiterals.Contains(p));
             Assert.AreEqual(2, clauseCollection[0].NegativeLiterals.Count);
-            Assert.AreEqual(q, clauseCollection[0].NegativeLiterals[0]);
-            Assert.AreEqual(r, clauseCollection[0].NegativeLiterals[1]);
+            Assert.IsTrue(clauseCollection[0].NegativeLiterals.Contains(q));
+            Assert.IsTrue(clauseCollection[0].NegativeLiterals.Contains(r));
 
             Assert.AreEqual(1, clauseCollection[1].PositiveLiterals.Count);
-            Assert.AreEqual(s, clauseCollection[1].PositiveLiterals[0]);
+            Assert.IsTrue(clauseCollection[1].PositiveLiterals.Contains(s));
             Assert.AreEqual(1, clauseCollection[1].NegativeLiterals.Count);
-            Assert.AreEqual(t, clauseCollection[1].NegativeLiterals[0]);
+            Assert.IsTrue(clauseCollection[1].NegativeLiterals.Contains(t));
         }
     }
 }
```

Comment:
No newline at the end of the file, how would it look if I tried to cat in on my terminal?

Reply:
![](https://thatlinuxbox.com/blog/images/articles/cat-meme-your-console_1.png)

Reply:
like that